server/readme.md

```diff
@@ -1,3 +1,70 @@
-# Book Api
+# Book Api​
 
-> uh oh.. I have bugs
\ No newline at end of file
+Create a dynamic library type application that stores books and allows you to add books. It should have a home page add page and books page. You are able to add a title, author, url, rating, and button to add the book.
+
+## Dependencies
+
+- Python
+  - [python](https://www.python.org/)
+- Flask
+  - [flask-pypi](https://pypi.org/project/Flask/)
+  - [flask-docs](https://flask.palletsprojects.com/en/1.1.x/)
+- Flask-SQLAlchemy
+  - [flask-sqlalchemy-pypi](https://pypi.org/project/Flask-SQLAlchemy/)
+  - [flask-sqlalchemy-docs](https://flask-sqlalchemy.palletsprojects.com/en/2.x/)
+- Flask-Marshmallow
+  - [flask-marshmallow-pypi](https://pypi.org/project/flask-marshmallow/)
+  - [flask-marshmallow-docs](https://flask-marshmallow.readthedocs.io/)
+- Marshmallow-SQLAlchemy
+  - [marshmallow-sqlalchemy-pypi](https://pypi.org/project/marshmallow-sqlalchemy/)
+  - [marshmallow-sqlalchemy-docs](https://marshmallow-sqlalchemy.readthedocs.io/en/latest/)
+- Flask-Bcrypt - [flask-bcrypt-pypi](https://pypi.org/project/Flask-Bcrypt/) - [flask-bcrypt-docs](https://flask-bcrypt.readthedocs.io/)
+  ​
+
+## Getting Started
+
+- download zip or clone
```

Comment:
Please let the user know they need to be in the server folder before install the dependencies.  If they're in the root folder we will be making our environment in the wrong spot.